### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by jesuarva**
# Description

Start writing project's documentation.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] No test needed.

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My code has been reviewed by at least one peer
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,89 @@
...
+> Detailed docs with installations and running instructions on [Backend.md](https://github.com/Lambda-School-Labs/CS10-Developer-Profiles/blob/master/backEnd/Backend.md)
+
+### Location:
+
+> [./backEnd/](https://github.com/Lambda-School-Labs/CS10-Developer-Profiles/tree/master/backEnd)
+
+# Frontend APP
+
+> Detailed docs with installations and running instructions on [Frontend.md](https://github.com/Lambda-School-Labs/CS10-Developer-Profiles/blob/master/backEnd/Frontend.md)
+
```
Comment (d-vail) [inline README.md]: Change link: https://github.com/Lambda-School-Labs/CS10-Developer-Profiles/blob/Documentation/front-end/Frontend.md

File: README.md [hunk 2]
```diff
@@ -1,8 +1,89 @@
...
+### Location:
+
+> [./front-end/lambda-in/](https://github.com/Lambda-School-Labs/CS10-Developer-Profiles/tree/master/front-end/lambda-in)
+
+### Components structure
+
+Componets prefixed with '_**@**_', belongs to _Material Design_
+
```
Comment (d-vail) [inline README.md]: Typo: Components

File: backEnd/Backend.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+```
+  $ cd backend/
+```
+
+lastly, install dependencies and run the server:
+
+```
+  $ yarn
+  ... isntalling dependencies
+  ...
```
Comment (d-vail) [inline backEnd/Backend.md]: Typo: installing dependencies

File: README.md (hunk 1 above)
Comment (jesuarva) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (jesuarva) [reply README.md]: done

File: backEnd/Backend.md (hunk 3 above)
Comment (jesuarva) [reply backEnd/Backend.md]: done.
I'll try to be more careful writing, thank you.

**Pull request merged**